infogami/infobase/core.py

```diff
@@ -83,7 +83,7 @@ def _process(self, value):
         if isinstance(value, list):
             return [self._process(v) for v in value]
         elif isinstance(value, dict):
-            return web.storage((k, self._process(v)) for k, v in iteritems(value))
+            return web.storage((k, self._process(v)) for k, v in sorted(iteritems(value)))
```

Comment:
```suggestion
            return web.storage((k, self._process(v)) for k, v in sorted(items(value)))
```
Drop the legacy nomenclature because putting it inside a sorted() call removes the difference.

Reply:
Oh, interesting! Well, in my latest push, I removed the call to `sorted()`, so I'll ignore this for now, but let me know if you still think it's relevant.

Reply:
`iteritems()` without `sorted()` is best.